src/resources/codemirror.ts

```diff
@@ -0,0 +1,9 @@
+// @ts-ignore
+import _CodeMirror from "codemirror";
+// @ts-ignore
+import _codeMirrorCss from "codemirror/lib/codemirror.css";
+import "codemirror/mode/yaml/yaml";
```

Comment:
how big are these files ? Is it ok to bundle all synax up in 1 file ?

Reply:
Yeah, they are small